examples/https-termination/README.md

```diff
@@ -8,7 +8,7 @@ reference a Secret in a different Namespace.
 
 ## 1. Deploy NGINX Gateway Fabric
 
-1. Follow the [installation instructions](/docs/installation.md) to deploy NGINX Gateway Fabric.
+1. Follow the [installation instructions](https://docs.nginx.com/nginx-gateway-fabric/installation/) to deploy NGINX Gateway Fabric.
```

Comment:
```suggestion
1. Follow the [installation instructions]({{< relref "/installation/" >}}) to deploy NGINX Gateway Fabric.
```
Swapping this to a `relref`: if someone clones the repo, any "production" URLs will direct them online when browsing/editing documentation offline.

Reply:
Will a `relref` work here? This doc is not in the `site` folder.

Reply:
I was commenting on why relrefs are useful as a follow-up to Alan, but relrefs don't work for documents not in the /site/content path. So no, a relref won't work for the link above.

It's easy to miss that we have docs in the github repo in the /docs path that don't go in the docs.nginx.com website